benches/benches.rs

```diff
@@ -12,6 +12,7 @@
 // limitations under the License.
 
 #![feature(test)]
+#![cfg_attr(not(feature = "cargo-clippy"), allow(unknown_lints))]
```

Comment:
Because we have many #[allow] for clippy rules but without `feature = cargo-clippy`. See https://github.com/pingcap/tikv/issues/3248

Reply:
Isn't all allows are specified via cmd args now?

Reply:
Any updates?

Reply:
It's probably best to have lints like so:

```rust
#![cfg_attr(feature = "cargo-clippy"), allow(this_random_lint, another_random_lint)]
```

It lets people run `cargo clippy` instead of needing to specificly run `make clippy`.